raiden/raiden_service.py

```diff
@@ -270,15 +267,33 @@ def __init__(
         self.gas_reserve_lock = gevent.lock.Semaphore()
         self.payment_identifier_lock = gevent.lock.Semaphore()
 
+        # Flag used to skip the processing of all Raiden events during the
+        # startup.
+        #
+        # Rationale: At the startup, the latest snapshot is restored and all
+        # state changes which are not 'part' of it are applied, the criteria to
+        # re-apply the state changes is their 'absence' in the snapshot, /not/
+        # their completeness. Because these state changes are re-execute
+        # in-order and some of their side-effects will already have been
+        # completed, the events should be delayed until the state is
+        # synchronized (e.g. an open channel state change, which has already
+        # been mined).
+        #
+        # Uncompleted events, i.e. the ones which don't have their side-effects
+        # applied, will be executed once the blockchain state is synchronized
+        # because of the node's queues.
+        self.ready_to_process_events = False
+
     def start(self):
         """ Start the node synchronously. Raises directly if anything went wrong on startup """
-        if not self.stop_event.ready():
-            raise RuntimeError(f'{self!r} already started')
+        assert self.stop_event.ready(), f'Node already started. node:{self!r}'
```

Comment:
Added `node:{self!r}` to all assert messages to make it easy to understand what is going on the CI